### PR TITLE
Update all dependencies

### PR DESCRIPTION
**Pull request by dwaynebailey**
Update all currently out of date dependencies to their latest released version.

Note:

1. I'm not addressing ranges vs pinning etc
2. If no pin existed I've added one
3. If a pin existed I've kept the dependency pinned even if it could have been made a range
4. If a range existed I've adjusted it as needed.

### CODE REVIEW

File: requirements/base.txt [hunk 1]
```diff
@@ -20,7 +20,7 @@ diff-match-patch==20121119
 dirsync==2.2.2
 elasticsearch>=1.0.0,<2.0.0
 jsonfield==1.0.3
-lxml>=3.0,<=3.6.0
+lxml>=3.0,<=3.6.4
```
Comment (unho) [inline requirements/base.txt]: I would bump the minimum to 3.5.0 to align with TTK.

File: requirements/base.txt [hunk 2]
```diff
@@ -23,7 +23,7 @@ jsonfield==1.0.3
 lxml>=3.0,<=3.6.4
 python-dateutil==2.5.3
 python-levenshtein==0.12.0
-rq>=0.5.0,<=0.5.6
+rq>=0.5.0,<=0.6.0
```
Comment (unho) [inline requirements/base.txt]: This maximum was pinned to prevent some bug. See daf4f345 for more information.

Comment (unho) [inline requirements/base.txt]: This maximum was pinned to prevent some bug. See daf4f345 for more information.

File: requirements/base.txt [hunk 3]
```diff
@@ -3,27 +3,27 @@
...
 cssmin==0.2.0
 diff-match-patch==20121119
 dirsync==2.2.2
 elasticsearch>=1.0.0,<2.0.0
 jsonfield==1.0.3
-lxml>=3.0,<=3.6.0
+lxml>=3.0,<=3.6.4
 python-dateutil==2.5.3
 python-levenshtein==0.12.0
-rq>=0.5.0,<=0.5.6
```
Comment (unho) [inline requirements/base.txt]: This maximum was pinned to prevent some bug. See daf4f345 for more information.

**Comment by unho** [comment]
lgtm

**Comment by ta2-1** [comment]
I have checked `rq v0.6.0` and it works fine. We use private methods of `Job` class they hasn't changed. So it's ok to pin the next `rq` version.

**Pull request merged**